src/rpdk/core/test.py

```diff
@@ -119,6 +121,40 @@ def get_overrides(root, region_name, endpoint_url):
     return overrides
 
 
+# pylint: disable=R0914
+def get_inputs(root, region_name, endpoint_url, value):
+    inputs = []
+    if not root:
+        return None
+
+    path = root / INPUTS
+    if not os.path.isdir(path):
+        return None
+
+    file_prefix = INPUTS + str(value) + "_"
+
+    directories = os.listdir(path)
+    if len(directories) > 0:
+        for file in directories:
+            if file_prefix in file:
```

Comment:
```suggestion
            if file.startswith(file_prefix):
```
Nitpick, but I think that matches the prefix semantic better

Reply:
I can go ahead and make this change.